packages/markdown/remark/src/shiki.ts

```diff
@@ -129,6 +131,10 @@ export async function createShikiHighlighter({
 	};
 }
 
+function normalizeMaybeArray(value: Properties[string]): string | null {
+	return Array.isArray(value) ? value.join(' ') : (value as string);
```

Comment:
If `value` is cast to `string`, when does the function return `null`?

Reply:
Updated @ematipico